docs/podman-create.1.md

```diff
@@ -654,6 +654,40 @@ change propagation properties of source mount. Say `/` is source mount for
 To disable automatic copying of data from the container path to the volume, use
 the `nocopy` flag. The `nocopy` flag can be set on bind mounts and named volumes.
 
+**--volumes-from**[=*[CONTAINER[:OPTIONS]]*]
+
+Mount volumes from the specified container(s).
+The `OPTIONS` are a comma delimited list and can be:
```

Comment:
*OPTIONS* is a placeholder, not a literal, so I think you want:

```markdown
The *OPTIONS* are a...
```

or:

```markdown
*OPTIONS* is a...
```

here.

Also, maybe "and can be:" -> "with the following available elements:" or some such.

Reply:
Fixed